src/Types/Transaction.purs

```diff
@@ -77,6 +77,8 @@ module Types.Transaction
 
 import Prelude
 
+import Contract.Numeric.Rational (Rational)
```

Comment:
More `Contract` imports. I can just remove it

Reply:
How does it always creep in :o